kubernetes/deployment.tmpl

```diff
@@ -112,6 +112,56 @@ spec:
               value: staging
             - name: REDIS_URI
               value: "redis://hamlet-staging-redis"
+          volumeMounts:  # ??? Is this necessary?
+            - name: static-assets
+              mountPath: "/static"
+
+        - name: hamlet-staging-nginx
+          image: zooniverse/apps-nginx:xenial
+          ports:
+            - containerPort: 80  # ??? could this conflict with the Django app?
+          volumeMounts:
+            - name: static-assets
+              mountPath: "/static"
+      volumes:
+        - name: static-assets
+          configMap:  # ??? use EmptyDir: {} instead of the ConfigMap?
+            name: hamlet-nginx-conf-staging
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: hamlet-nginx-conf-staging
+data:
+  nginx.conf: |+
+    upstream docker-hamlet {
+      server localhost:81;
+    }
+
+    server {
+      server_name hamlet-staging.zooniverse.org
+      include /etc/nginx/ssl.default.conf;
+      gzip_types *;
+
+      location ~ ^/static/ {
+        root /hamlet/static/;
+        gzip_static on; # to serve pre-gzipped version
+        expires max;
+        add_header Cache-Control public;
+      }
+
+      location / {
```

Comment:
This means... for every OTHER request, let `http://docker-hamlet` handle it. Which I assume is the Django app.